setup.sh

```diff
@@ -247,7 +145,8 @@ function _docker_container
   then
     ${CRI} exec "${USE_TTY}" "${CONTAINER_NAME}" "${@:+$@}"
   else
-    # If no container yet, run a temporary one: https://github.com/docker-mailserver/docker-mailserver/pull/1874#issuecomment-809781531
+    # if no container is running, run a temporary one:
```

Comment:
Was that by accident (putting the URL in a separate line prefixed by two spaces)

Reply:
Yes:)